modules/manage/pages/cluster-maintenance/disk-utilization.adoc

```diff
@@ -250,7 +250,7 @@ A periodic housekeeping task in Redpanda performs compaction and removes data th
 * When `retention_local_strict` is false (default), the housekeeping process removes data above the configured consumable retention. This means that data usage is allowed to expand to occupy more of the log data reservation.
 * When `retention_local_strict` is true, the housekeeping process uses local retention settings to select what data to remove.
 +
-NOTE: The `retention_local_strict` property is set to true in clusters upgraded from release 23.1 and earlier.
+NOTE: The `retention_local_strict` property is set to false when upgrading a cluster from release 23.1 and earlier.
```

Comment:
The behavior didn't change for `retention_local_strict` but it did change for `space_manage_enable` property.

Reply:
I'm not following -- the PR said:

```Make the following configuration options have these default values regardless of the upgrade history of the cluster

space_management_enable=true
retention_local_strict=false```

I want to confirm what you're saying -- even though the default is `false` for new clusters, it will remain set to `true` for upgraded clusters?